go.mod

```diff
@@ -146,5 +147,7 @@ require (
 	google.golang.org/genproto v0.0.0-20231211222908-989df2bf70f3 // indirect
 	google.golang.org/genproto/googleapis/api v0.0.0-20231211222908-989df2bf70f3 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20231212172506-995d672761c0 // indirect
-	gopkg.in/yaml.v3 v3.0.1 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	gopkg.in/square/go-jose.v2 v2.6.0
```

Comment:
We should not include go-jose.v2, we should use `go.step.sm/crypto` if possible, alternativelly we can also use `github.com/go-jose/go-jose/v3`

Reply:
Not relying on it directly anymore, but it's still indirect through https://github.com/coreos/go-oidc/.